docs/source/multi_node.md

```diff
@@ -0,0 +1,29 @@
+# Distributed GPU Training
+
+Calling `submit_to_azure_if_needed` with `num_nodes>1` will prepare the AzureML run_configuration for distributed training jobs using IntelMpi ([https://docs.microsoft.com/en-us/azure/machine-learning/how-to-train-distributed-gpu](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-train-distributed-gpu)) with `process_count_per_node=1` (the default, for per-node launch) and `node_count=num_nodes`.
+
+After calling `submit_to_azure_if_needed` with `num_nodes>1`, then call: `set_environment_variables_for_multi_node()` to ensure the environment variables for Mpi are set correctly.
```

Comment:
To be more precise, this sets the environment variables that Pytorch Lightning expects, based on the env variables that AzureML/mpi prepare.